.cirrus.yml

```diff
@@ -129,7 +129,7 @@ precommit_task:
 #
 # Testing (Linux)
 #
-test_task:
+task:
```

Comment:
I know this is what the Cirrus warning suggests, but it makes me uncomfortable because it is not really good YAML -- because you probably shouldn't use duplicate mapping keys.  
For details, see [this discussion](https://github.com/SciTools/iris-grib/pull/283#issuecomment-874570061) ...

So I'd love to find a way that **_doesn't_** use a repeated key, if we could ?

Reply:
Oddly, that's precisely what we had before... it's almost like `cirrus-ci`s YAML parser is complaining unnecessarily

Reply:
So it seems like YAML, but YAML according to `cirrus-ci` :thinking:

Reply:
> Oddly, that's precisely what we had before

I'm sure they changed it.

> YAML according to cirrus-ci

My reading, yes.
I couldn't work out whether they are actually **_using_** https://github.com/cirruslabs/yaml.
If so not sure why, as they don't seem to have changed anything significant.  Perhaps it is just a "pin".
It seems that the [parent original](https://github.com/go-yaml/yaml) has removed the API that let you use it in this way, but they are [reconsidering whether to reintroduce something similar](https://github.com/go-yaml/yaml/pull/484)

Reply:
I agree, as Cirrus recommend this, we have to believe in it.
https://cirrus-ci.org/guide/writing-tasks/